### ISSUE TITLE
Bootloader.hex too big (RBL Nano)

### ISSUE THREAD

**Issue by sk1tt1sh** [RedBearLab/nRF51822-Arduino#45]
I'm trying to follow the arduino instructions. I have flashed the RBL driver to the mk20 usb device (hold button plug in drag RBL_nRF51822_MK20.bin to it) and now I am trying to put bootloader.hex on it for arduino programming and it says "File is too large".

When attmpting to upload from arduino ide it doesn't connect: "Connecting to programmer: .avrdude: butterfly_recv(): programmer is not responding" I have installed the CDC serial driver and selected the correct com port.

Thanks for your help!

**Comment by Cheong2K** [comment]
Download the zip file https://github.com/RedBearLab/nRF51822-Arduino/archive/S130.zip
And unzip it and load the bootloader.hex to the mbed drive again to see.

**Comment by sk1tt1sh** [comment]
Thank you. I managed to upload a custom built firmware (s120) on the nano. I will give this a whirl if I get a chance and let you know, but I learned how to write the code and merge it.

As a side note, I was unable to successfully do anything with windows. I had to use ubuntu vm with device filter to catch the MK20 plug in and the sudo cp -xv merged.hex /media/$USER/MBED/

Thanks for a fun product! :)
